### ISSUE TITLE
Map key syntax highlighting issues

### ISSUE THREAD

**Issue by bgriffith** [atom/language-sass#148]
Possibly related to #146 

When you have maps inside maps, all the keys bar the first are incorrectly highlighted. In this example, they take the highlighting of the valid colours.

The code:
```sass
$custom-palettes: (
    alr: (
        alr-blue: (
            x-light: rgb(240, 243, 246),
            light: rgb(144, 165, 187),
            mid-light: rgb(88, 122, 152),
            base: rgb(1, 51, 102),
            dark: rgb(11, 25, 41),
        ),
        alr-orange: (
            base: rgb(218, 125, 0),
            dark: rgb(191, 108, 0),
            x-dark: rgb(125, 73, 5),
        ),
        alr-green: (
            base: rgb(169, 176, 36),
            dark: rgb(103, 106, 55),
        ),
        alr-grey: (
            xx-light: rgb(222, 222, 222),
            x-light: rgb(200, 200, 200),
            light: rgb(68, 68, 68),
            base: rgb(38, 38, 38),
            dark: rgb(17, 17, 17),
        ),
        alr-white: (
            base: rgb(255, 255, 255),
        ),
        alr-black: (
            base: rgb(0, 0, 0),
        ),
    ),
);
```

The issue:

![screen shot 2016-07-07 at 12 21 17](https://cloud.githubusercontent.com/assets/2854338/16651657/59b538b8-443d-11e6-90f4-e816bbffafdb.png)

**Comment by esdoppio** [comment]
Should be fixed in #149. Thanks for the report! 👍

**Comment by bgriffith** [comment]
@esdoppio Awesome. Thanks for the fix 👍
